flakytests/registration/test_header

```diff
@@ -0,0 +1 @@
+../lib/bash/test_header
```

Comment:
Oh, interesting. Never noticed how symlinks are displayed in GitHub diff UI.

Here's my local copy:

![image](https://user-images.githubusercontent.com/304786/63663342-22314200-c816-11e9-8239-a544169eec8b.png)

So hopefully created the symlink correctly.